src/ucp/core/ucp_request.inl

```diff
@@ -94,6 +96,7 @@
             } \
         } else { \
             __req = ((ucp_request_t*)(_param)->request) - 1; \
+            __req->flags |= UCP_REQUEST_FLAG_EXTERNAL; \
```

Comment:
align by `=`

Reply:
aligned

Reply:
need to clear this flag (or init to 0) if it is internal request

Reply:
ok, set to 0

Reply:
where? Also why do you do `|=`? User may not touch UCX part of request at all, so req->flags may contain garbage

Reply:
re-pushed